Rocket.Chat/Models/User.swift

```diff
@@ -28,6 +28,8 @@ class User: BaseModel {
         get { return UserStatus(rawValue: privateStatus) ?? UserStatus.offline }
         set { privateStatus = newValue.rawValue }
     }
+
+    var utcOffset: Double?
```

Comment:
Needs to add `@objc dynamic` here also, this way we can save this information in database.

Reply:
Swift doesn't let me @objc a Double 🤷‍♂️ Any ideas?